src/pages/informationmodel-details-page/information-model-details-page.jsx

```diff
@@ -26,11 +26,18 @@ const renderJSONSchema = schema => {
 
   return (
     <div>
-      <pre style={{ whiteSpace: 'pre' }}>{JSON.stringify(schema, null, 2)}</pre>
+      <pre style={{ whiteSpace: 'pre-wrap', overflowWrap: 'break-word' }}>
+        {JSON.stringify(schema, null, 2)}
+      </pre>
     </div>
   );
 };
 
+const renderRdfData = rdfData => (
+  <pre style={{ whiteSpace: 'pre-wrap', overflowWrap: 'break-word' }}>
+    {rdfData && rdfData}
```

Comment:
And (&&) check does not make sense inside pre element. Move it outside so that you have rdfData && (<pre>something</pre>)